src/test/java/it/gov/pagopa/ranking/connector/rest/pdv/PdvErrorDecoderExt.java

```diff
@@ -0,0 +1,22 @@
+package it.gov.pagopa.ranking.connector.rest.pdv;
+
+import feign.Response;
+
+public class PdvErrorDecoderExt extends PdvErrorDecoder{
+
+    private static int invocationCount;
```

Comment:
use a thread safe type instead
```suggestion
    private static AtomicInteger invocationCount;
```